include/simdjson/inline/parsedjson_iterator.h

```diff
@@ -235,12 +243,18 @@ dom::parser::Iterator::Iterator(const dom::parser &pj) noexcept(false)
     depth_index[depth].scope_type = current_type;
   }
 }
-
+///Os3
 dom::parser::Iterator::Iterator(
     const dom::parser::Iterator &o) noexcept
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR. Please remove the ///Os3 comment.

Reply:
Are you sure? This is copy constructor.

Reply:
You are correct, the copy constructor is fine, what is not find is the cryptic comment just above it.

Reply:
comment removed